apps/condo/domains/common/components/Table/Renders.tsx

```diff
@@ -89,79 +90,62 @@ export const renderMeterReading = (values: string[], resourceId: string, measure
     return `${getIntegerPartOfReading(values[0])} ${measure}`
 }
 
-const fillMoneyWithSpaces = (substring: string, startIndex: number, spaces: Array<number>) => {
-    const chars = []
-    for (let i = 0; i < substring.length; i++) {
-        const absoluteIndex = startIndex + i
-        const char = substring.charAt(i)
-        if (spaces.includes(absoluteIndex)) {
-            chars.push(MONEY_PARTS_SEPARATOR)
+const getCurrencySymbol = (currencyCode) => {
+    return (0).toLocaleString(
+        undefined,
+        {
+            style: 'currency',
+            currency: currencyCode,
+            minimumFractionDigits: 0,
+            maximumFractionDigits: 0,
+            currencyDisplay: 'narrowSymbol',
         }
-        chars.push(char)
-    }
-    return chars.join('')
+    ).replace(/\d/g, '').trim()
 }
 
-const recolorMoney = (
-    text: string, startIndex: number, separatorIndex: number, spaces: Array<number>, extraStyles: React.CSSProperties
-) => {
-    if (separatorIndex === -1 || startIndex + text.length <= separatorIndex) {
-        return (
-            <Typography.Text style={extraStyles}>
-                {fillMoneyWithSpaces(text, startIndex, spaces)}
-            </Typography.Text>
-        )
-    }
-    if (startIndex >= separatorIndex) {
-        return (
-            <Typography.Text type={'secondary'} style={extraStyles}>
-                {fillMoneyWithSpaces(text, startIndex, spaces)}
-            </Typography.Text>
-        )
-    }
-    const blackLength = separatorIndex - startIndex
-    return (
-        <>
-            <Typography.Text style={extraStyles}>
-                {fillMoneyWithSpaces(text.substring(0, blackLength), startIndex, spaces)}
-            </Typography.Text>
-            <Typography.Text type={'secondary'} style={extraStyles}>
-                {text.substring(blackLength)}
-            </Typography.Text>
-        </>
-    )
-}
+const dimText = (text: string, extraStyles: React.CSSProperties = {}) => (
+    <Typography.Text type={'secondary'} style={extraStyles}>
+        {text}
+    </Typography.Text>
+)
 
-export const getMoneyRender = (search: string, currencyMark = '₽', partSeparator = DEFAULT_CURRENCY_SEPARATOR) => {
+const renderMoney = (currencyValuePart: string, currencyDecimalPart: string, currencySymbol: string) => (
+    <>{currencyValuePart}{dimText(',' + currencyDecimalPart)}{NBSP}{currencySymbol}</>
```

Comment:
"," ?

Reply:
You should better use `Intl.formatNumber`, then find index of `['.', ',']`, then find digits after....and use calculated index to recolour string

Reply:
Or may be I'm a bit confused about this logic....Can you provide more detailed explanation what's going on?

Reply:
input: `2400.21000000`

1) Use formatNumber: `// P2,400.21`
2) Replace P with '' `// 240.21`
3) Replace "," with " " `// 2 400.21`
4) Replace "." with "," `// 2 400,21`
5) Find decimal part and real part of the number
6) Recolor the decimal part